app/templates/partials/questions/repeatinganswer.html

```diff
@@ -28,14 +26,23 @@ <h3 class="neptune">{{ _('Person ') }}<span class="js-household-loopindex">{{ lo
         </h3>
         <fieldset class="question__fieldset">
           <legend class="question__legend u-vh">{{ _('Person ') }} <span class="js-household-loopindex">{{ loop.index }}</span></legend>
-          {% for answer in composite_answer %}
+          {% for input in name_form %}
             {% with render_guidance = False %}
+              {% set schema_item = question.answers[loop.index0] %}
```

Comment:
I think an input is more appropriate here, as it's the WTForms input which doesn't have the same name as the answer in the schema.